packages/flutter/test/material/toggle_buttons_test.dart

```diff
@@ -665,6 +665,124 @@ void main() {
     expect(material.type, MaterialType.button);
   });
 
+  testWidgets('Custom button fillColor - Non MaterialState', (WidgetTester tester) async {
+    Material buttonColor(String text) {
+      return tester.widget<Material>(find.descendant(
```

Comment:
return tester.widget<Material>(
        find.descendant(
          of: find.byType(RawMaterialButton),
          matching: find.widgetWithText(Material, text),
        ),
      );